File: src/main/java/org/isisdis/common/EventSuggestion.scala
package org.isisdis.common

import java.util.Date
import scala.collection.mutable.ListBuffer

class EventSuggestion {
  var startDate : Date = null
  var endDate : Date = null
	var description : String = null
	var contacts  = new ListBuffer[Long]()
  var collaborator : Int = 0
  var address : String = null
  
  def this(startDate: Date, endDate : Date, description : String, contacts : ListBuffer[Long], collaborator : Int, address : String ) {
    this()
    this.startDate = startDate
    this.endDate = endDate
    this.description = description
    this.contacts = contacts
    this.collaborator = collaborator
    this.address = address
  }
  
  
}

File: src/main/java/org/isisdis/common/EventSuggestion.java
package org.isisdis.common;

import java.util.Date;
import java.util.Set;

import lombok.Data;

public @Data class EventSuggestion {
	private Date startDate;
	private Date endDate;
	private String description;
	private Set<Contact> contacts;
	private Collaborator collaborator;
	private String address;
}
